app/lib/import_export.rb

```diff
@@ -301,6 +301,8 @@ def extract_patients_details(patients, fields)
       patient_details[:expected_purge_ts] = patient.expected_purge_date_exp if fields.include?(:expected_purge_ts)
       patient_details[:full_status] = patient.status&.to_s&.humanize&.downcase if fields.include?(:full_status)
       patient_details[:status] = patient.status&.to_s&.humanize&.downcase&.sub('exposure ', '')&.sub('isolation ', '') if fields.include?(:status)
+      patient_details[:primary_language] = Languages.translate_code_to_display(patient.primary_language)
```

Comment:
actually, can we separate languages out into a separate type in `PATIENT_FIELD_TYPES` under `import_export_constants.rb` and then on line 292 of `import_export.rb`, we can add something like:
```
      (fields & PATIENT_FIELD_TYPES[:languages]).each { |field| patient_details[field] = Languages.translate_code_to_display(patient[field]) }
```

Reply:
Resolved and rebased.